packages/server/utils/MattermostServerManager.ts

```diff
@@ -91,6 +92,29 @@ class MattermostServerManager extends MattermostManager {
   constructor(webhookUrl: string) {
     super(webhookUrl)
   }
+
+  static async getBestWebhook(userId: string, teamId: string, dataLoader: DataLoaderWorker) {
```

Comment:
-1 opt to remove & put this logic in the datalaoder. dataloaders should use managers, but try to avoid managers using dataloaders